services/openai.py

```diff
@@ -20,7 +19,9 @@ def get_embeddings(texts: List[str]) -> List[List[float]]:
         Exception: If the OpenAI API call fails.
     """
     # Call the OpenAI API to get the embeddings
-    response = openai.Embedding.create(input=texts, model="text-embedding-ada-002")
+    response = await openai.Embedding.create(
```

Comment:
I believe it should be [acreate](https://github.com/openai/openai-python/blob/main/openai/api_resources/embedding.py#L55) for async calls.